app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt

```diff
@@ -226,6 +229,14 @@ class SellPayloadPresenter(
     }
 
     override fun cashOut() {
+        if (sellInteractor.isNeedShowInformDialog()) {
+            view?.navigateToInformationScreen()
+        } else {
+            navigateToMoonpayWidget()
+        }
+    }
+
+    override fun navigateToMoonpayWidget() {
```

Comment:
presenter doesn't navigate anywhere, it can buildMoonpayWidget or prepare it, but not navigate